runners/core-java/src/main/java/org/apache/beam/runners/core/SimpleDoFnRunner.java

```diff
@@ -1074,6 +1075,12 @@ public Timer withOutputTimestamp(Instant outputTimestamp) {
       return this;
     }
 
+    @Override
+    public Timer withOutputTimestampOffset(Duration outputTimestampOffset) {
+      this.outputTimestampOffset = outputTimestampOffset;
+      return this;
+    }
```

Comment:
I'm not sure that we need withOutputTimestampOffset - I think withOutputTimestamp is sufficient.

Reply:
@reuvenlax  done